### ISSUE TITLE
Home Notices Sorta Broken

### ISSUE THREAD

**Issue by Daniel99j2** [Nulkem/betterseqta#22]
Most notices work, but sometimes if they have a table/bold text this happens...
<img width="744" alt="Screenshot 2023-04-06 082615" src="https://user-images.githubusercontent.com/121533735/230242746-0d431012-ffc1-4385-a130-3ab03e502b66.png">

**Comment by Daniel99j2** [comment]
Should look like this:
<img width="270" alt="Screenshot 2023-04-06 082919" src="https://user-images.githubusercontent.com/121533735/230243112-cd71b5a8-e303-4bd4-af3c-0dab7050e9f7.png">

**Comment by proJM-Coding** [comment]
I can back this up. When the text had a background colour added it started to overlap. 
Also in the home section it show notices which I have selected not to see. This is still a very great extension, keep up the good work.

What it looks like with better SEQTA:
![Screenshot 2023-04-12 6 32 17 PM](https://user-images.githubusercontent.com/81658610/231408995-dca1555e-545c-4fef-8f6f-0ae2830f1b98.png)
What is should look like:
![Screenshot 2023-04-12 6 33 24 PM](https://user-images.githubusercontent.com/81658610/231408982-df3b11cf-d08b-4e83-96dd-15538c59fdfe.png)

**Comment by Crazypersonalph** [comment]
I don't think this project is still being actively developed, there hasn't been any commits since October last year.

**Comment by AustinAggiss** [comment]
I agree, the author most likely was in Year 12 and finished his studies. This would revoke his access to SEQTA for testing. Its unfortunate but the only cause I can think of.

**Comment by Crazypersonalph** [comment]
I made a fork of the project, we can make new commits there

Although we don't have ownership of the extension, or the domain, we can still continue development

https://github.com/Crazypersonalph/betterseqta

**Comment by proJM-Coding** [comment]
Is is possible to make another extention with updates?

**Comment by Crazypersonalph** [comment]
> Is is possible to make another extention with updates?

We would need to request Nulkem to transfer ownership of the extension so that we can make changes to the extension that everybody already has installed

**Comment by proJM-Coding** [comment]
That would be good but how much would it cost to put one on the google web store?

**Comment by Crazypersonalph** [comment]
> That would be good but how much would it cost to put one on the google web store?

$5 to sign up for a Google Chrome Store Developer account.

**Comment by norangeflame** [comment]
> Is is possible to make another extention with updates?

You can always turn dev mode on in chrome://extensions, and then import the SRC folder with any updates you may have. The extension will work like normal but only be available to you. This method costs nothing. The SRC folder being the folder with SEQTA.js, background.js, etc.
However, the average user may find this a bit complicated and so this is obviously wouldn't be an easy solution for the majority of betterseqta users.

**Comment by proJM-Coding** [comment]
The problem is my school locked dev mod off and I'm not sure if they would unblock another extention. They don't like doing that much.

**Comment by norangeflame** [comment]
> The problem is my school locked dev mod off and I'm not sure if they would unblock another extention. They don't like doing that much.

Our school rents out their computers to us and so we don't have administrator access and can't do things like install extensions. Originally they hadn't blocked other versions of Chrome such as Chrome Dev and Chrome Canary, and I could still use extensions but now they are blocked too. As of right now I don't have a solution for this. 

However, if your school's IT department is really strict on computer usage, and you really want to bypass the restrictions, you can try either installing an OS on a USB and booting to that, or using a Windows Installation USB to create partitions on your drive, and then install a second OS on your main drive. Contact norangeflame#9782 on Discord if you want to know more (this is a temporary acc and will be removed soon)

**Comment by Crazypersonalph** [comment]
> > The problem is my school locked dev mod off and I'm not sure if they would unblock another extention. They don't like doing that much.
> 
> Our school rents out their computers to us and so we don't have administrator access and can't do things like install extensions. Originally they hadn't blocked other versions of Chrome such as Chrome Dev and Chrome Canary, and I could still use extensions but now they are blocked too. As of right now I don't have a solution for this.
> 
> However, if your school's IT department is really strict on computer usage, and you really want to bypass the restrictions, you can try either installing an OS on a USB and booting to that, or using a Windows Installation USB to create partitions on your drive, and then install a second OS on your main drive. Contact @norangeflame_2#9782 on Discord if you want to know more (this is a temporary acc and will be removed soon)

We are going a bit off-topic, but what about using other browsers such as Firefox and Brave, not just Chrome?

**Comment by Crazypersonalph** [comment]
I would just like to mention that there is some major refactoring to do with the BetterSeqta code, the code base is kind of a mess
If you can help maintain the project with me, it would be greatly appreciated
https://github.com/Crazypersonalph/betterseqta

**Comment by Crazypersonalph** [comment]
With the original issue, I am suspecting that it is an issue with the code around Line 3016 in Seqta.js
It is where notices are loaded and the CSS is changed.

**Comment by norangeflame** [comment]
> I would just like to mention that there is some major refactoring to do with the BetterSeqta code, the code base is kind of a mess
> If you can help maintain the project with me, it would be greatly appreciated
> https://github.com/Crazypersonalph/betterseqta

I would be happy to help however I am not very well versed in javascript. I’ll have a look though.

**Comment by norangeflame** [comment]
> > > The problem is my school locked dev mod off and I'm not sure if they would unblock another extention. They don't like doing that much.
> > 
> > 
> > Our school rents out their computers to us and so we don't have administrator access and can't do things like install extensions. Originally they hadn't blocked other versions of Chrome such as Chrome Dev and Chrome Canary, and I could still use extensions but now they are blocked too. As of right now I don't have a solution for this.
> > However, if your school's IT department is really strict on computer usage, and you really want to bypass the restrictions, you can try either installing an OS on a USB and booting to that, or using a Windows Installation USB to create partitions on your drive, and then install a second OS on your main drive. Contact @norangeflame_2#9782 on Discord if you want to know more (this is a temporary acc and will be removed soon)
> 
> We are going a bit off-topic, but what about using other browsers such as Firefox and Brave, not just Chrome?

Yeah it is a bit off topic sorry, but I’ve tried FireFox which didn’t work however you are correct about Brave (at least for my situation), it doesn’t block extensions.

**Comment by norangeflame** [comment]
> Should look like this: <img alt="Screenshot 2023-04-06 082919" width="270" src="https://user-images.githubusercontent.com/121533735/230243112-cd71b5a8-e303-4bd4-af3c-0dab7050e9f7.png">

@Daniel99j2 Would you be able to Inspect Element and then copy the code for both the original notice and BetterSEQTA broken notice (take any sensitive details out) and paste it. One of my broken notices contained a heap of empty HTML "p" tags that aren't in the original notice. 
I've taken a look at the BetterSEQTA code, and I think I've found the issue (maybe).

**Comment by Crazypersonalph** [comment]
Let's move discussion over to https://github.com/crazypersonalph/betterseqta/
